en-US/Grammar.xml

```diff
@@ -606,11 +872,56 @@ Split contractions and abbreviations into separate paragraphs. -->
 		</formalpara>
 
 	</section> 
-	 <section id="gender-references">
-		<title>Gender References</title>
+	 <section id="pronouns-gender-references">
+		<title>Pronouns and Gender References</title>
+		 <para>
+			Do not use gender-specific pronouns in documentation, except to refer to a specific named user, such as in a case study. It is far less awkward to read a sentence that uses "they" and "their" rather than "he/she" and "his/hers". It is acceptable to use "they" to refer to one person, with a plural verb. 
+		</para>
+		 <para>			
+			In most cases, when giving instructions, use the imperative mood or use "you". In more general explanations, you can use "users" or "new users". Do not use "one" in place of "you" when writing technical documentation. Using "one" is far too formal. Do not use "it" to refer to a person.
```

Comment:
Do not use "one" in place of "you" when writing technical documentation because it is too formal.

Reply:
Fixed